src/mixins/wallet.js

```diff
@@ -49,7 +50,7 @@ export default {
     },
     async fetchWalletBalance() {
       const balance = await getAccountBalance(this.getAddress);
-      this.balance = Math.ceil(Number(balance)).toLocaleString('en-US');
+      this.balance = formatNumber(Number(balance));
```

Comment:
The updated collect modal has been used the `balance` for calculation, so it is better to return the balance in number and handle the formatting at the very end